.github/workflows/continous-integration.yaml

```diff
@@ -0,0 +1,130 @@
+name: Continous Integration
+
+# This workflow is triggered on pushes to the repository.
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Test
+    strategy:
+      matrix:
+        os:
+          - windows-latest
+          - ubuntu-latest
+          - macos-latest
+        node-version: [12]
+        include:
+          - os: ubuntu-latest
+            node-version: 10
+          - os: ubuntu-latest
+            node-version: 14
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Update NPM
+        run: npm i -g npm
```

Comment:
IIRC, we are using the `npm` version bundled with the Node.js version we are testing on, to ensure we support older npm versions too. Is there any particular reason you want to run our CI tests using the latest `npm` version?